tools/bump-lts-line.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+version_input=$1
+
+if [ -z "$version_input" ]; then
+  echo "Please enter a version to bump, e.g. 2.452"
+  exit 1
+fi
+
+git checkout stable-${version_input}
+file="profile.d/stable"
+last_line=$(tail -n 1 $file)
+version=$(echo $last_line | cut -d'=' -f2)
+IFS='.' read -ra version_parts <<< "$version"
+version_parts[2]=$((version_parts[2] + 1))
+
+if [ "${version_parts[2]}" -eq 4 ]; then
+  echo "Warning: You are about to bump from .3 to .4. Are you sure you want to continue?"
+  read
+fi
+
+new_version="${version_parts[0]}.${version_parts[1]}.${version_parts[2]}"
+
+sed -i '' -e '$d' $file
```

Comment:
This seems to be specific to macOS.  [Stack overflow](https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux) suggests this change.  That change works for me with Ubuntu 22.04
```suggestion
sed -i'' -e '$d' $file
```